sources/web/build.sh

```diff
@@ -29,12 +29,22 @@ WEB_DIR=$BUILD_DIR/web/nb
 
 mkdir -p $WEB_DIR
 
+# Experimental UI build step
+cd datalab/polymer
+bower install
+tsc
+polymer build
+rsync -avpq ./build/experimental/ ../static/experimental
```

Comment:
This line is redundant (the copy is performed again below with the rest of the copy commands)